Source/Extensions/Blazorise.Charts.Zoom/ChartZoomAdapter.cs

```diff
@@ -0,0 +1,49 @@
+﻿#region Using directives
+using System;
+using System.Threading.Tasks;
+using Microsoft.JSInterop;
+#endregion
+
+namespace Blazorise.Charts.Zoom;
+
+/// <summary>
+/// Middleman class for the ChartZoom plugin.
+/// </summary>
+public class ChartZoomAdapter<TItem>
```

Comment:
Why the adapter?

The one method can simply go to the razor.cs file. (??)

Reply:
It can, yes. But I wanted to prevent the public methods that are meant for internal use from leaking to the users. Remember, we have some public methods in other parts of the code that have a comment "internal use only". This is to prevent that.